stm32-data-gen/src/interrupts.rs

```diff
@@ -502,7 +502,7 @@ fn valid_signals(peri: &str) -> Vec<String> {
         ("GTZC", &["GLOBAL", "ILA"]),
         ("WWDG", &["GLOBAL", "RST"]),
         ("USB_OTG_FS", &["GLOBAL", "EP1_OUT", "EP1_IN", "WKUP"]),
-        ("USB_OTG_HS", &["GLOBAL", "EP1_OUT", "EP1_IN", "WKUP"]),
+        ("USB_OTG_HS", &["GLOBAL", "EP1_OUT", "EP1_IN", "WKUP", "USB"]),
```

Comment:
this is not quite right. If the chip has a single interrupt for USB, it should get the signal name "GLOBAL". Reasons:
- "USB" as a signal name is redundant because it's already known it's an interrupt for the USB peripheral.
- consistency between all families.